packages/ChangesReporting/Contract/Output/OutputFormatterInterface.php

```diff
@@ -4,11 +4,17 @@
 
 namespace Rector\ChangesReporting\Contract\Output;
 
+use Rector\Core\Contract\Rector\RectorInterface;
 use Rector\Core\ValueObject\ProcessResult;
 
 interface OutputFormatterInterface
 {
     public function getName(): string;
 
     public function report(ProcessResult $processResult): void;
+
+    /**
+     * @param RectorInterface[] $rectors
+     */
```

Comment:
This should be independent on process command
The interface should be moved to a new class.